ffs/terms.md

```diff
@@ -0,0 +1,61 @@
+
+## Picking up a Funded Proposal
+A second contractor may pick up any proposal that is in an 'Expired' state. He/she must take note of the current terms and notify the community of their interest in picking up the proposal. If there are any adjustments to the proposal that need to be made to fit individual needs or schedules, he/she will bring the ammended terms to the community for review and approval.
+
+In the event that the community agrees to the ammends, the contractor must submit a thorough update ON the proposal (meaning having to make a PR request to the website GitHub) with the terms that are changed at the bottom, as well as the words 'Proposal Ammended / New Contractor' at the top in bold. This will tell the community that the terms have been ammended and to take special note of the changes at the bottom. If the PR is not merged and the website rebuilt and uploaded in a timely manner, it is not the fault of the contractor, and the PR is sufficient proof of compliance.
+
+In the event that the community does not agree to the new terms, the contractor can make further ammends and repeat the process or choose to not pick up the proposal. He/she may, of course, also choose to do the work as he/she pleases without community input, though they cannot formally charge the FFS. In the event that he/she completes the project on his/her own the community may decide to release funds anyway, but there are no guarentees.
+
+## Investigation
+In the event that a contractor does not comply with the terms above, a special Monero investigative team will be put together that should include one Core team member (or trusted proxy) and at least one other volunteer. The proposal is considered 'on hold' until investigation is complete. Should work continue and milestones be completed during a time of investigation, no payouts are to be made until the investigation is complete.
+
+The contractor is expected to comply with the investigative team's requests, which may include, but are not limited to, chat logs, proof of work, and other requested materials.
+
+The investigative team is write up their conclusion and present to the community, who can then either provide a formal dismissal or allow the contractor to continue the contract.
+
+## Appeal
+In the event that a contract leaves a proposal or is dismissed, but he/she has, in the time that followed, proven their goodwill, they can make an appeal to the community to resume the contract, assuming another contractor hasn't already picked up the proposal. They can provide whatever evidence they think will be helpful in convincing the community to let them resume their previous work.
+
+Should the community decide to allow the contractor to resume his/her work, he/she may begin immediately. Should they decide not to, the contractor is expected not to be a baby about it.
+
+## Conclusion (a note about the community)
```

Comment:
I think this entire conclusion could be re-written into a disclaimer, and put in the beginning.